osu.Game.Rulesets.Catch/CatchRuleset.cs

```diff
@@ -93,6 +93,8 @@ public override IEnumerable<Mod> GetModsFor(ModType type)
 
         public override string Description => "osu!catch";
 
+        public override string ShortName => "fruits";

```

Comment:
Is this temporary? Shouldn't it be "catch"? Given that this is not used right now, why don't we name it that immediately? Will avoid future database stuff.

Reply:
the web still uses `fruits` so that's what goes here. may change going forward, but isn't such a straight-forward change.